hypertrace-gradle-publish-maven-central-plugin/src/main/java/org/hypertrace/gradle/publishing/HypertracePublishMavenCentralExtension.java

```diff
@@ -0,0 +1,53 @@
+package org.hypertrace.gradle.publishing;
+
+import org.gradle.api.Action;
+import org.gradle.api.NamedDomainObjectContainer;
+import org.gradle.api.model.ObjectFactory;
+import org.gradle.api.plugins.ExtensionAware;
+import org.gradle.api.provider.Property;
+
+import javax.inject.Inject;
+
+public abstract class HypertracePublishMavenCentralExtension implements ExtensionAware {
+  private static final String DEFAULT_URL = "https://www.hypertrace.org";
+  public final Property<String> url;
+  public final Property<String> scmConnection;
+  public final Property<String> scmDeveloperConnection;
+  public final Property<String> scmUrl;
+    public final NamedDomainObjectContainer<PomLicense> licenses;
```

Comment:
indentation

Also, this + developers is a container - do we expect multiple licenses (maybe irrelevant given my other comments)

Reply:
indentation fixed.
It now support only one license.